grunt_tasks/webpack.config.js

```diff
@@ -69,7 +70,14 @@ module.exports = {
             'window.jQuery': 'jquery'
         }),
         // Disable writing the output file if a build error occurs
-        new webpack.NoEmitOnErrorsPlugin()
+        new webpack.NoEmitOnErrorsPlugin(),
+        new GoogleFontsPlugin({
```

Comment:
I'm not sure how to refer to the built css file from `global.styl`.  I assume there is some `@import` stylus statement, but the path to reach the `googlefonts.css` file in `static/built` is eluding me.